custom.ini

```diff
@@ -9,7 +9,8 @@ provisioning = {absolute-path-to-cwd}/.grafana/provisioning
 temp_data_lifetime = 24h
 
 # Directory where grafana can store logs
-logs = {absolute-path-to-cwd}/.grafana/log
+# Unneeded here as it is specified when grafana is started
+# logs = {absolute-path-to-cwd}/.grafana/log
```

Comment:
if we don't need this configuration we should remove it entirely. I had problems previously on Linux as the regular user didn't have write access to the default log directory with installed grafana. Not sure if this is the reason that things are breaking with this patch on Ubuntu.

Reply:
FYI: Removing this part of the change fixes the breakage on Ubuntu ...